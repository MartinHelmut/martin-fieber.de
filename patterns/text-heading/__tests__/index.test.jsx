import "jest-styled-components";
import React from "react";
import renderer from "react-test-renderer";

import TextHeading from "../index";

describe("TextHeading", () => {
  test("should render", () => {
    const tree = renderer
      .create(<TextHeading>Small text heading</TextHeading>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
