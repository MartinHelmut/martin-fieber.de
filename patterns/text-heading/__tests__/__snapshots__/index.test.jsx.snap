// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextHeading should render 1`] = `
<h4
  className="text-heading__TextHeading-jmogz4-0 jtlteb"
>
  Small text heading
</h4>
`;
